src/raven.js

```diff
@@ -168,6 +170,7 @@ Raven.prototype = {
             this._isRavenInstalled = true;
         }
 
+        Error.stackTraceLimit = this._globalOptions.stackTraceLimit;
```

Comment:
Yeah, this only affects V8, and unfortunately, it's a global option.

V8 just defaults to a limiting, 10 frames. We've personally hit an issue with this in Sentry since React generates some long stack traces.

In theory, we'd push all browsers to capture `Infinity` frames, then `slice()` out of that, but that's not how reality works. :(